docs/guides/general/delta_live_tables.md

```diff
@@ -0,0 +1,134 @@
+# :material-table-heart: Working with Delta Live Tables
+
+<img src="https://img.shields.io/badge/available%20since-0.8.0-green?style=for-the-badge" alt="Available since 0.8.0"/>
+
+`dbx` provides capabilities to deploy, launch and reference pipelines based on the [Delta Live Tables framework](https://docs.databricks.com/workflows/delta-live-tables/index.html).
+
+!!! warning "Development support for DLT Pipelines"
+
+    Please note that `dbx` doesn't provide interactive development and execution capabilities for DLT Pipelines.
+
+## :material-hexagon-multiple-outline: Providing pipeline definition in the deployment file
+
+Example pipeline definition would look like this:
+
+```yaml title="conf/deployment.yml"
+environments:
+  workflows:
+    - name: "sample-dlt-pipeline"
+      workflow_type: "pipeline" #(1)
+      storage: "dbfs:/some/location" #(2)
+      configuration: #(3)
+        "property1": "value"
+        "property2": "value2"
+      clusters: #(4)
+        - label: "some-label" #(5)
+          spark_conf:
+            "spark.property1": "value"
+            "spark.property2": "value2"
+          aws_attributes:
+            ...
+          instance_pool_id: "instance-pool://some-pool"
+          driver_instance_pool_id: "instance-pool://some-pool"
+          policy_id: "cluster-policy://some-policy"
+          autoscale:
```

Comment:
Hey! I tried with the new example; everything else worked perfectly except the autoscale. Curious why?

ValidationError: 1 validation error for Deployment
workflows -> 0 -> Pipeline -> clusters -> 0 -> autoscale
  Model values must be a dict; you may not have returned a dictionary from a root validator (type=type_error)

Reply:
fixed in the later commits, thanks!